docs/concepts/what-is-a-temporal-sdk.md

```diff
@@ -36,6 +36,8 @@ The following SDKs are in alpha/pre-alpha development stages, but are not yet su
 
 ### Third-party SDKs
 
-The following third party SDKs exist, but are not supported in the application development guide:
+The following third-party SDKs exist but are not supported in the [Application development guide](/application-development):
 
-- [coinbase/temporal-ruby](https://github.com/coinbase/temporal-ruby)
+- [Scala](https://github.com/vitaliihonta/zio-temporal) from [@vitaliihonta](https://github.com/vitaliihonta)
+- [Python](https://github.com/firdaus/temporal-python-sdk) from [@firdaus](https://github.com/firdaus)
```

Comment:
This one isn’t maintained afaik and I don’t see a reason to mention it at this point